Add `osu!mania 4K World Cup 2025: Semifinals Recap` news post (#13739)
ppy/osu-wiki: news/2025/2025-09-20-osumania-4k-world-cup-2025-semifinals-recap.md

## Semifinals summary

| Team A |  |  | Team B |
| --: | :-: | :-: | :-- |
| Vietnam ::{ flag=VN }:: | 1 | **6** | ::{ flag=ID }:: **Indonesia** |
| **South Korea** ::{ flag=KR }:: | **6** | 1 | ::{ flag=PE }:: Peru |
| **Hong Kong** ::{ flag=HK }:: | **6** | 0 | ::{ flag=PH }:: Philippines |
| Canada ::{ flag=CA }:: | 3 | **6** | ::{ flag=TH }:: **Thailand** |
| **United States** ::{ flag=US }:: | **6** | 0 | ::{ flag=CL }:: Chile |
| **South Korea** ::{ flag=KR }:: | **6** | 2 | ::{ flag=ID }:: **Indonesia** |
| Hong Kong ::{ flag=HK }:: | 2 | **6** | ::{ flag=TH }:: **Thailand** |
| **China** ::{ flag=CN }:: | **6** | 1 | ::{ flag=GB }:: United Kingdom |

> It's easy to see from the table above that only the best of the best teams are still standing. At this point in the tournament, matches are less likely to be decided by sweeping differences. And even if we've already seen quite a few close matches on prior rounds, this is soon going to be changing starting from this round onwards. Look no further than this week's [score highlights](#score-highlights)!

Reviewer: if this is saying that sweeps are less likely, then it seems to contradict the table above that's mostly filled with sweeps. if the second sentence is about close matches being prevalent before but *not* from here on, then that contradicts the first sentence. i recommend reducing this to the main point, after figuring out what it is <.<

Reviewer: smth about "while the results here seem like sweeps, many of the points in these matches were decided by very close score margins, which is something we've seen more in these later tournament rounds"

Reviewer: ```suggestion
While the results here seem like sweeps, many of the points in these matches were decided by incredibly close score margins, which is something we've been seeing more and more in these later tournament rounds. To appreciate those individual score differences, we've [compiled the highlights below](#score-highlights)!
```

Before that though, we want to highlight potentially the closest match we've had during the weekend. Potentially the closest match thus far as well!

### ::{ flag=CA }:: Canada vs. ::{ flag=TH }:: Thailand

**[You can view the match information here!](https://osu.ppy.sh/community/matches/119259424)**\
**[You can view the match VOD here!](https://www.twitch.tv/videos/2565463262?t=0h7m23s)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/VIDEO_ID?start=XXX&end=YYY" frameborder="0" allowfullscreen></iframe>
</div>

> Both teams had good reason to be nervous about this match. With an 80% pickem prediction rate, the favourite was definitely ::{ flag=TH }:: Thailand, a powerhouse team that had an unfortunate early drop into the losers' bracket. They had a taste of the podium for the first time last year, and came into the tournament looking to repeat — or even surpass! — that feat. But ::{ flag=CA }:: Canada was not going to be a pushover, coming in hot straight from the winners' bracket, being defeated by the #1 seed China.

Reviewer: ```suggestion
Both teams had good reason to be nervous about this match. With an 80% pick'em prediction rate, the favourite was definitely ::{ flag=TH }:: Thailand, a powerhouse team that had an unfortunate early drop into the losers' bracket. They had a taste of the podium for the first time last year, and came into the tournament looking to repeat (or even surpass!) that feat. Nevertheless, ::{ flag=CA }:: Canada was not going to be a pushover, coming in hot straight from the winners' bracket, only sent down by the #1 seed of the tournament, China.
```

But there is an exception, [Camellia - Dans la mer de son (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2432008#mania/5298381) (HB2), plenty of famous players failed to keep full combo on this map, only a few players achieved the perfect. Among whom ::{ flag=CN }:: [DawnX](https://osu.ppy.sh/users/8534840) got the highest 995,201 score, this impressive score is extremely outstanding on the whole round.

# Interview

<style>
    .news-chat-quote__avatar {
        float: left;
        width: 40px;
        height: 40px;
        border-radius: 50%;
        margin-left: -50px;
    }

    .news-chat-quote__text-container {
        margin-left: 50px;
    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-no-group {
        color: #FFFFFF;
    }

    .news-chat-quote__colour-fa {
        color: #00FFFF;
    }
</style>

![](/wiki/shared/news/2025-09-20-osumania-4k-world-cup-2025-semifinals-recap/Marmalade_butcher.png)

> As a bonus, we got to interview [Marmalade butcher](https://osu.ppy.sh/beatmaps/artists/220) who was very eager to get interviewed by the community. Their commitment to their work really shone through this World Cup, and seeing them this excited to see the community's reactions got them pumped up!

Reviewer: the last sentence didn't quite make sense

```suggestion
As a bonus, we got to interview [Marmalade butcher](https://osu.ppy.sh/beatmaps/artists/220), composer of *SIX BILLION MNSN* for the Semifinals LN3 slot. Their commitment to their work really shone through this World Cup, and seeing the community reaction seems to only have pumped them up!
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10365024">Sakura006</a></p>

How were you made aware of osu! prior to this commission? What were your initial impressions seeing users mapping and playing your tracks in a rhythm game?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/beatmaps/artists/220" style="background-image: url('/wiki/shared/avatars/Marmalade_butcher.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-fa" href="https://osu.ppy.sh/beatmaps/artists/220">Marmalade butcher</a></p>

I've never actually played it myself, but I knew about osu! beforehand through social media and such, where I learned that Marmalade butcher's songs seem to be quite popular in this rhythm game. I never imagined we'd get an offer to make a song for the World Cup, though.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10365024" style="background-image: url('/wiki/shared/avatars/Sakura006.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10365024">Sakura006</a></p>

> Do you believe mapping in rhythm games is a good medium to re-interpret music through gameplay? Do you think this was achieved with SIX BILLION MNSN?

Reviewer: ```suggestion
Do you believe mapping in rhythm games is a good medium to reinterpret music through gameplay? Do you think this was achieved with SIX BILLION MNSN?
```

Lastly, would you say this has been a good experience thus far? From the commissioning process to seeing the results of everyone's hard labour, what have been your favourite parts of the process?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/beatmaps/artists/220" style="background-image: url('/wiki/shared/avatars/Marmalade_butcher.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-fa" href="https://osu.ppy.sh/beatmaps/artists/220">Marmalade butcher</a></p>

It was a lot of fun! I'd love to write songs again anytime! I even think it'd be amazing to do a live show worldwide through osu! someday! I hope everyone looks forward to that day!

Honestly, it doesn't really sink in while I'm making it or right after I submit it, but seeing everyone actually play it, enjoy it, and get hyped up—that's the best part. Every single one of them has passion poured into it! (I always pour passion into them!) If you keep playing and listening to the songs even after the World Cup ends, that would make me even happier!

</div>

---

> So many questions, and only one way to find out the answer. Make sure to tune in to the Finals weekend on the [osulive](https://www.twitch.tv/osulive) livestream for the next chapter of the osu!mania World Cup! Want to see all the stats for yourself, then be sure to go to our [wiki](/wiki/Tournaments/MWC/2025_4k). We hope to see you in the chat, and enjoy the Finals weekend!

Reviewer: ```suggestion
So many questions, and only one way to find out the answer. Make sure to tune in to the Finals matches later today on the [osulive](https://www.twitch.tv/osulive) livestream for the next chapter of the osu!mania World Cup! Want to see all the stats for yourself? Be sure to check the [tournament information page](/wiki/Tournaments/MWC/2025_4k). We hope to see you in the chat, and enjoy the Finals weekend!
```
